cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env -S bash -e
+
+WORKSPACE="${WORKSPACE:-'.'}"
+
+if [ -d "$RELEASE_GPG_HOMEDIR" ]
+then
+  echo "Cleaning up GPG homedir..."
+  rm -rf "$RELEASE_GPG_HOMEDIR"
```

Comment:
I would prefer to use a gpg command to clean up the keys instead of working directly to the internal key directory:
> gpg --delete-secret-keys ${keyId}

Reply:
Why, though? This way is simpler (we don't need to know the key ID) and safer (we're sure to delete absolutely everything).

Reply:
because the key is cached for some time, meanwhile someone else can sign stuff with it

Reply:
Ok, I had to bring everything together under a single script (in order to remember the ID of the imported key), but I managed to do what you wanted.

And I think it's better now, since we only have to call a single script when releasing; see hibernate/hibernate-search#2856 .

WDYT?